src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
Do we consider the option that the Ball obj can be created without state, like a transparent ball?
I suppose that this constructor should be removed.